### DISCUSSION TITLE
Notice to assigners

### DISCUSSION THREAD

**Discussion by pyropig** [jtr13/cc21fall2#74] [General]
@Abhimanyu210100 @Gangadharareddy514 
In the list of assignees github put branch owner themselves in the list, you may want to check if there's accidental assignment back to themselves on previous pulls. See discussion on this pull https://github.com/jtr13/cc21fall2/pull/22
![1](https://user-images.githubusercontent.com/31974777/140439624-0b3626b6-90bf-4617-b281-c7ee911c3778.PNG)
![2](https://user-images.githubusercontent.com/31974777/140439626-0e0d3aed-9be8-4c46-82cd-c923c6add9b7.PNG)

**Comment by Abhimanyu210100** [comment]
Hi @pyropig. Thank you for pointing this out, I will correct any mistakes that have been there and keep them in mind next time
